wilds/datasets/globalwheat_dataset.py

```diff
@@ -94,14 +94,14 @@
     'VLB',
     'VSC',
     'Wad Medani',
+    'Eschikon'

```

Comment:
Are we adding new data?

Reply:
no, it was an other typo mistake in the metadata. Eschikon is the place where ETHZ data was acquired